smithy-aws-protocol-tests/model/restJson1/empty-input-output.smithy

```diff
@@ -83,20 +83,28 @@ apply EmptyInputAndEmptyOutput @httpRequestTests([
         protocol: restJson1,
         method: "POST",
         uri: "/EmptyInputAndEmptyOutput",
-        body: "{}",
+        body: "",
         bodyMediaType: "application/json"
     },
 ])
 
 apply EmptyInputAndEmptyOutput @httpResponseTests([
     {
         id: "RestJsonEmptyInputAndEmptyOutput",
-        documentation: "Empty output serializes no payload",
+        documentation: "Empty output deserializes no payload",
```

Comment:
not an issue, just a clarification: these tests are about serializing requests and responses. They aren't meant to be oriented towards clients or servers, but rather just about how messages are serialized. So a service should be able to take these tests and make sure the service is implemented correctly. That's why I used the word serialized here instead of deseralize.

Reply:
Thanks for clarifying that. Updating this one thinking it was a copy/paste bug. Can revert this back.